### ISSUE TITLE
Multi-mic roast audio capture (MCP-owned, config-driven) — record per roast for FC training data (v1: capture-only)

### ISSUE THREAD

**Issue by syamaner** [syamaner/coffee-roaster-mcp#176]
Turn every roast into FC training data + model-validation material. Surfaced by roast 4 (the audio detector missed; #175). **v1 is capture-only** — no in-system annotation (Label Studio stays the labeler), no fine-tune.

## Scope (v1)
- **The MCP captures roast audio**, config-driven (it already owns the audio device for detection, so it owns capture). Read the device **once** and tee: feed the FC inference + write the WAV(s) from the same stream (no double-open, no extra device contention).
- **Output per roast:** one mono WAV per channel + a **session JSON** carrying `recording_started_monotonic` and, in **recording-relative seconds**, the model-FC and the per-window **confidence timeline** (#175). All timestamps recording-relative — the MCP owns both the recording and the detection clock, so the mapping is sample-accurate with **no sync tone** needed.
- **No annotation in-system.** The operator labels FC start+end in **Label Studio** post-roast (existing `coffee-first-crack-detection` flow). The RoastPilot **FC button stays a pure human override** for live control — no FC-end button, no annotation duality, minimal UI.

## Config (`coffee-roaster-mcp.yaml`)
```yaml
recording:
  autocapture: true                    # capture every roast, no button
  export_location: ~/roasts/captures   # default, configurable
  device: <capture device>             # macOS aggregate OR the 2-ch interface
  channels: [0, 1]                     # channel → mic map
  sample_rate: 44100
```
Plus optional `start_capture` / `stop_capture` MCP commands; **config-first** is the minimum (zero agent/UI work to capture).

## Hardware / cross-platform device (operator decision)
Use a **single class-compliant 2-channel USB audio interface** (both mics → one device): enumerates as one multi-channel, **sample-locked** device on **both macOS and Linux/Pi**, which kills the macOS-CoreAudio-aggregate-vs-Pi-ALSA split and the 2-separate-USB-mic drift problem. Abstract it as `device` + `channels` in config.

**ACCEPTANCE GATE — Pi-5 CPU feasibility (conditional):** the single-interface choice is conditional on the Pi 5 having headroom to capture 2 channels **while running the torch-free FC inference** (D27 is already tight — `onnx_threads: 2`). Verify capture + inference together stay within budget on the Pi before committing the hardware; if not, fall back (1-channel capture, or capture-only-on-Mac for now).

## Out of scope (revisit later)
Auto-annotation from the live FC mark; the operator-FC-vs-MCP-FC **validation-over-time** dashboard (a follow-on once data accrues); fine-tune / retrain automation. v1 just produces the labeled-able WAVs + the MCP-FC/confidence logs.

## Privacy / storage
WAVs are large and **never committed** — land them in `export_location` (default `~/roasts/captures`), gitignored. Reuse the `coffee-first-crack-detection` WAV-naming + annotation-JSON conventions so Label Studio + `convert_labelstudio_export` / `propagate_annotations` / `chunk_audio` ingest them unchanged.

## Links
roastpilot-agent #134 (roast 4 — the miss that motivates this), #175 (FC confidence observability — same data story, do first), `coffee-first-crack-detection` (`scripts/record_mics.py`, `docs/multi_mic_setup.md`, the data-prep pipeline this feeds), D42 (the corpus). Composes with the FC-timing work (#174 T0, #170 FC onset).

**Comment by syamaner** [comment]
v1 shipped in **0.1.9**: config-driven roast audio capture — mono + **multi-device (option A, independent streams)** + record-check + set_recording_metadata + record_mics-compatible naming. Both mics captured on roast 6. The follow-on work is tracked as its own issues: **#180** (overflow fix, done/pending-0.1.10), **#181** (full-roast lifecycle — recorder currently stops at FC), **#178** (live-levels mic check). Closing the v1 delivery.
